src/ship_cmd.cpp

```diff
@@ -174,26 +192,26 @@ static void CheckIfShipNeedsService(Vehicle *v)
 		return;
 	}
 
-	uint max_distance;
+	uint max_penalty;
```

Comment:
Distance or cost, not penalty. A penalty is applied to a cost or distance, and the cost or distance is checked against.

Reply:
But everywhere it's called max_penalty. If this is wrong, then every other places are wrong too.